### PR TITLE
better randomness

### PR DESCRIPTION
**Pull request by mcmenaminadrian**
Better randomness at last - this is a small change so feel free to merge it if you are happy

### CODE REVIEW

File: SplitFiles/game.c [hunk 1]
```diff
@@ -82,8 +82,9 @@ int main_game()
     printf(PRINTAT "\x01\x07" "\x12\x31\x10\x32" "Can you handle the pressure?\x12\x30\x10\x30");
     printf(PRINTAT "\x05\x0B" "Press any key to start");
     wait_for_a_key(NULL, NULL);
-    int rando = *((int *)49161);
-    srand(rando);
+    // get a random seed based on frame count
+    int *rando = 23672;
```
Comment (ConorMcPersonal) [inline SplitFiles/game.c:86]: I don't think this is going to work as you're hoping, not with startup=1 anyway.  I don't see the frame count being updated when I write test code to print it out.  In the three positions for the timer I instead see 22, 0, 0 - I suspect 22 is the frame count needed before the C runtime takes over the machine...

We could check the keyboard asynchronously in a loop until the users presses a key - count the iterations and use that as the seed?

Comment (mcmenaminadrian) [reply SplitFiles/game.c:86]: I did test it and it seemed to work

Comment (mcmenaminadrian) [reply SplitFiles/game.c:86]: let me check again

Comment (mcmenaminadrian) [reply SplitFiles/game.c:86]: Definitely giving me monotonically increasing numbers as I go on with more and more runs, though the first time I run it it always gives me 22 but it doesn't just repeat the sequence on a reload.

Comment (ConorMcPersonal) [reply SplitFiles/game.c:86]: How are you re-running - by hitting "(R)un" on the command line in Fuse?  If so my guess is: the update is coming from dropping back to Sinclair Basic mode, but once you hit run it won't change again until the C program stops running.

Comment (mcmenaminadrian) [reply SplitFiles/game.c:86]: yes, that's possible. But that doesn't matter except for the fact that the seed for the first game will always be the same unfortunately.

Comment (mcmenaminadrian) [reply SplitFiles/game.c:86]: I can do your key thing though

Comment (mcmenaminadrian) [reply SplitFiles/game.c:86]: OK see other PR for this now!

**Pull request closed without merging**